developers.rst

```diff
@@ -25,6 +25,9 @@ transliteration too.
 Permissions History
 -------------------
 
+- Cheryl Sabella was given push privileges on February 19, 2019 by Brett Cannon,
+  on the recommendation of Victor Stinner.
```

Comment:
It would be nice to add @terryjreedy as well to the recommendation.

Reply:
Ooops, sorry @terryjreedy: fixed in commit 8b85ad54f22d85e5fae30fd91e8eac75e6f433b4.

Reply:
really sorry @terryjreedy it's my fault :/ I was not sure about the owner of the mentor.

Reply:
Carol, thank you for noticing. Others, no apologies needed.  I just glad to finally have her fully onboard.

Reply:
Congrats Cheryl!